specification/security/resource-manager/Microsoft.Security/preview/2023-07-01-preview/integrations.json

```diff
@@ -0,0 +1,670 @@
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "delete": {
+        "x-ms-examples": {
+          "Delete an integration over subscription scope": {
+            "$ref": "./examples/integrations/DeleteIntegration_example.json"
+          },
+          "Delete an integration over security connector scope": {
+            "$ref": "./examples/integrations/DeleteSecurityConnectorIntegration_example.json"
+          },
+          "Delete an integration over management group scope": {
+            "$ref": "./examples/integrations/DeleteManagementGroupIntegration_example.json"
+          }
+        },
+        "tags": [
+          "Integrations"
+        ],
+        "description": "Delete an integration over a given scope",
+        "operationId": "Integrations_Delete",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/Scope"
+          },
+          {
+            "$ref": "#/parameters/IntegrationId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success - Integration was deleted",
+            "headers": {
+              "location": {
+                "type": "string",
+                "description": "Location URL for the deletion status"
+              }
+            }
+          },
+          "202": {
+            "description": "Accepted - The operation still in progress"
+          },
+          "204": {
+            "description": "No Content - Integration does not exist"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/{scope}/providers/Microsoft.Security/integrations/{integrationId}/users": {
+      "post": {
+        "x-ms-examples": {
+          "Get integration user names over subscription scope": {
+            "$ref": "./examples/integrations/GetIntegrationUserNames_example.json"
+          },
+          "Get integration user names over security connector scope": {
+            "$ref": "./examples/integrations/GetSecurityConnectorIntegrationUserNames_example.json"
+          },
+          "Get integration user names over management group scope": {
+            "$ref": "./examples/integrations/GetManagementGroupIntegrationUserNames_example.json"
+          }
+        },
+        "tags": [
+          "Integrations"
+        ],
+        "description": "Get user integration names for the requested scope by integrationId",
+        "operationId": "IntegrationUserNames_List",
```

Comment:
incorrect namespace `IntegrationUserNames` - suggested `Integrations_ListUserNames` to retain `Integrations_` prefix all the way